### DISCUSSION TITLE
Automatic formatting of possessive apostrophes for citations

### DISCUSSION THREAD

**Discussion by hgoers** [quarto-dev/quarto-cli#9751] [Feature Requests]
### Description

Hello, 

Can you please add a feature to Quarto's citations rendering that automatically formats possessive apostrophes? 

For example, it would be great if the following: `@smith2024's theory of X touches on Y.` rendered to produce: `Smith's (2024) theory of X touches on Y.`

I can get this output from the following: `Smith's [-@smith2024] theory of X touches on Y.` but it would be lovely if I did not have to type out author's name and citation key.

Thanks in advance!

**Comment by cscheid** [comment]
Unfortunately I think that's going to be hard for Quarto to do. Quarto doesn't actually directly control the generation of the citation text; we use Pandoc's processor ("citeproc"), which in turn uses CSL: https://citationstyles.org/

It might be possible to create a CSL file that causes the citations to render correctly in this case. I do think it's unlikely, because the typographical conventions and interaction with the grammatical rules of English are pretty intricate here.
